multi-repo-action/org-workflow-add.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/google/go-github/v42/github"
+	"golang.org/x/oauth2"
+)
+
+// *****SET THESE PARAMETERS*****
+const ORG_NAME string = "organization name"
+const PAT string = "access token"
+
+//Adds the Google Scorecard workflow to all repositores under the given organization
+func main() {
+	//Get github user client
+	context := context.Background()
+	tokenService := oauth2.StaticTokenSource(
+		&oauth2.Token{AccessToken: PAT},
+	)
+	tokenClient := oauth2.NewClient(context, tokenService)
+	client := github.NewClient(tokenClient)
+
+	//Get repositories under organization
+	lops := &github.RepositoryListByOrgOptions{Type: "all"}
+	repos, _, err := client.Repositories.ListByOrg(context, ORG_NAME, lops)
+	err_check(err, "List Org Repos Error")
+
+	//Convert to list of repository names
+	repoNames := []string{}
+	for _, repo := range repos {
+		repoNames = append(repoNames, *repo.Name)
+	}
+
+	//Get yml file into byte array
+	fileContent, _ := ioutil.ReadFile("scorecards.yml")
+
+	for _, repoName := range repoNames {
```

Comment:
Something to think about: if the PR fails for some reasons, the user will be left in a state where *some* repos only will have the PR created. How do we recover? How do we allow users to run the command but not create duplicated PRs for repos that already have it created?

Reply:
Made errors skip to next repo in iteration instead of doing os.Exit

Reply:
there's another case: repos where scorecard is not installed but you already sent a PR. How do you skip these repos?

Reply:
interesting tools I came across:
https://github.com/lindell/multi-gitter
https://cli.github.com/manual/gh_extension

Not saying we should these, just sending links for inspiration and fyi.

Reply:
Added checks so the repo gets skipped if the branch or workflow already exists. I didn't check for the PR existing because if the base branch doesn't exist, then the PR can't.